telemetry/specs/INT.mdk

```diff
@@ -1408,25 +1412,48 @@ INT-MX Header:
   can assign values in the range 0x0001 to 0xFFFF.
 
 * **DS Instruction (16b):** Instruction bitmap specific to the INT domain
-  identified by the *Domain Specific ID*. Each bit that is set requests that
-  metadata be sent to the monitoring system.
-
-  Some metadata can be defined as 'source-only' metadata by the INT domain.
-  Unlike other metadata, 'source-only' metadata is embedded in the packet by the
-  INT source node, as *Optional Domain Specific 'Source-Only' Metadata*. The
-  *DS Instruction* bits corresponding to 'source-only' metadata do not serve as
-  instructions for downstream INT nodes to insert their own metadata. Instead,
-  they indicate that the metadata is embedded, and that each node including
-  the source, transit, and sink nodes should report that source metadata to the
-  monitoring system along with the node's own metadata. Note that there are
-  two ways to include the 'source-only' metadata in the Telemetry Report,
-  described below.
+  identified by the *Domain Specific ID*. When a bit is defined for a particular
+  domain, in addition to the metadata semantics and syntax, the definition must
+  specify the behavior with respect to the following properties:
+
+    - **DS Instruction Mode:**
```

Comment:
Below two flag 'bits' (Export, Source-Inserted) are mentioned part of each DS instruction definition. Ppl may think these bits as included in either INT or telemetry headers, but they're not as I understand. Are we proposing these bits part of formal YANG model?
Wonder if we can explain them as 'properties' of each DS instruction that each domain has to decide and specify in some form, e.g., the YANG model.

Reply:
They are not described as "bits". It says "properties". And absolutely they should be included in the YANG model.